libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.kt

```diff
@@ -148,16 +148,23 @@ import java.util.function.Consumer
  * @Deprecated This class will no longer be public starting in Mobile SDK 13.0.  It
  * is no longer necessary to extend or change LoginActivity's instance of this class
  * to support multi-factor authentication.  If there are other uses cases please
- * inform the team via Github or our Trailblazer community.  
+ * inform the team via Github or our Trailblazer community.
  */
 @Deprecated(
     "This class will no longer be public starting in Mobile SDK 13.0.",
     level = DeprecationLevel.WARNING,
 )
 open class OAuthWebviewHelper : KeyChainAliasCallback {
 
+    /** The default, locally generated code verifier */
     private var codeVerifier: String? = null
 
+    /** For Salesforce Identity UI Bridge API support, an overriding front door bridge URL to use in place of the default initial URL */
+    var isUsingFrontDoorBridgeUrl = false
```

Comment:
NIT: The comment description makes this sound like it is a URL, not a boolean.  And does this or `frontDoorBridgeCodeVerifier` need to be public?

Reply:
Good cleanup for both points.  👉🏻 https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/2594/commits/74d1572e8da7bdaa6be98dc4eba9217f43531739 I feel like Android Studio has stopped suggesting access modifiers for some reason.  It used to be very aggressive about suggesting private or reductions in access when applicable.  🤨